people/0xPoe.toml

```diff
@@ -1,6 +1,6 @@
-name = 'Rustin'
-github = 'Rustin170506'
+name = 'Poe'
```

Comment:
So we should stop calling you Rustin, is that correct?

Reply:
You can call me Dongpo if you prefer. Ed told me that my Chinese name is not that hard. 😂